bigtable/tests/mutations_integration_test.cc

```diff
@@ -378,7 +388,7 @@ TEST_F(MutationIntegrationTest, DeleteFromColumnStartingFromTest) {
 
   // Create records
   CreateCells(*table, created_cells);
-  std::int64_t time_begin = 1000;
+  std::chrono::duration<std::int64_t> time_begin(1000);
```

Comment:
Ideally we would like to say something like:

```C++
using namespace std::chrono_literals; // assumes C++14
auto time_begin = 1ms;
```

And the library automatically knows to what units this needs to be converted inside.